canonical-web-and-design/canonicalwebteam.search
# Replace circle CI with github actions

## Done
Fixes https://github.com/canonical-web-and-design/canonicalwebteam.search/issues/18

## QA

Github actions should run successfully (linting, inclusive naming, tests, publish)
